do-app-platform-op-cli/README.md

```diff
@@ -1,314 +1,285 @@
+* [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl)
 
-### Step 1: Install 1Password and DigitalOcean tools
+> [!NOTE]
+> If you're using macOS or Linux, you only need to follow [Step 1: Install doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl) in the `doctl` installation guide.
 
-Install the following on your Mac, Windows or Linux machine:
+## Step 1: Set up your command-line tools
 
-- 1Password 8 for [Mac](https://1password.com/downloads/mac/) or [Windows](https://1password.com/downloads/windows/) or [Linux](https://1password.com/downloads/linux/)
-- [1Password CLI 2.9.0](https://developer.1password.com/docs/cli/get-started/#install) or later
-- [`doctl`](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl)
-  > **Note**
-  >
-  > **Only** [Step 1: Install doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl) in the `doctl` installation guide is required for this step for Mac and Linux. 
-
-### Step 2: Add your 1Password account and configure your DigitalOcean credentials.
-
-If you haven't already done so, add your 1Password account and connect 1Password CLI to your desktop app, then configure the DigitalOcean shell plugin:
+After you install 1Password CLI, add your 1Password account, then follow these steps:
 
 1. [Add your 1Password account](https://support.1password.com/add-account/) to 1Password 8 for Mac, Windows or Linux.
-2. [Connect 1Password CLI to the 1Password app](https://developer.1password.com/docs/cli/about-biometric-unlock#step-1-connect-1password-cli-with-the-1password-app).
-3. [Create a DigitalOcean personal access token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) with both read and write scopes.
-4. [Configure the DigitalOcean shell plugin](https://developer.1password.com/docs/cli/shell-plugins/digitalocean#step-1-configure-your-default-credentials) for Mac and Linux. Choose `Import into 1Password…` to save your DigitalOcean personal access token in your 1Password account and authenticate `doctl`.
-5. Windows users, follow the steps in [Step 3 Use the API token to grant account access to doctl](https://docs.digitalocean.com/reference/doctl/how-to/install/#step-1-install-doctl).
+2. [Connect 1Password CLI to the 1Password app](https://developer.1password.com/docs/cli/app-integration/).
+3. [Create a DigitalOcean personal access token](https://docs.digitalocean.com/reference/api/create-personal-access-token/) and select scopes for **read** and **write**.
+4. Grant access to `doctl`:
+	* If you're using macOS or Linux, [configure the DigitalOcean shell plugin](https://developer.1password.com/docs/cli/shell-plugins/digitalocean#step-1-configure-your-default-credentials) for 1Password CLI. Choose `Import into 1Password…` to save your DigitalOcean personal access token in your 1Password account and authenticate `doctl`.
```

Comment:
Order of the steps changed here, so depending on which way you wanna go:

```suggestion
	* If you're using macOS or Linux, [configure the DigitalOcean shell plugin](https://developer.1password.com/docs/cli/shell-plugins/digitalocean/#step-2-configure-your-default-credentials) for 1Password CLI. Choose `Import into 1Password…` to save your DigitalOcean personal access token in your 1Password account and authenticate `doctl`.
```

or

```suggestion
	* If you're using macOS or Linux, [configure the DigitalOcean shell plugin](https://developer.1password.com/docs/cli/shell-plugins/digitalocean/) for 1Password CLI. Choose `Import into 1Password…` to save your DigitalOcean personal access token in your 1Password account and authenticate `doctl`.
```